### PR TITLE
PAAS-1605: Replace 'mode' parameter with nodes counts and sizes

### PR DESCRIPTION
**Pull request by kevinvandel**
JIRA issue: https://jira.jahia.org/browse/PAAS-1605

:warning: to be merged with https://github.com/Jahia/jahia-cloud-modules/pull/345

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,19 +213,20 @@ The base JPS, called by Jahia Cloud to create an environment.
...
-| vaultSecretId    | Vault Role Id secret to use                                                                                                                                   |
+| parameter          | comment                                                                                  |
+|--------------------|------------------------------------------------------------------------------------------|
+| productVersion     | Jahia version (eg: `8.0.2.0`)                                                            |
+| rootpwd            | Jahia root password                                                                      |
+| toolspwd           | Jahia tools password                                                                     |
+| browsingNodeCount  | Number of Jahia browsing nodes                                                           |
+| browsingNodeSize   | Size of Jahia browsing nodes (`small`, `medium` or `large`)                              |
+| processingNodeSize | Size of Jahia processing node (`small`, `medium` or `large`)                             |
```
Comment (jahia-carp) [inline README.md:223]: missing haproxyNodeCount / sqldbNodeCount and sqldbNodeSize parameters

Comment (kevinvandel) [reply README.md:224]: fixed

**Review by jahia-carp — APPROVED** [review]

**Review by laurentfufu — APPROVED** [review]

**Review by laurentfufu — CHANGES_REQUESTED** [review]
XMX expressed like `[0-9].[0-9]G` will not work and tomcat will fail :(

**Comment by kevinvandel** [comment]
@laurentfufu XMX values fixed

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**